### ISSUE TITLE
Why Windows Only

### ISSUE THREAD

**Issue by bWildered1** [github0null/stm8-debug#7]
Nice job.

Just wondering what makes this 'Windows Only'?

Aren't VS code, sdcc, stm8flash, stm8-gdb, openocd, etc all multi-platform?

Take care

M

**Comment by github0null** [comment]
This plug-in support 'IAR' and 'COSMIC', For them, i only have Windows version gdb.

**Comment by MarrekNozka** [comment]
Yes, but stm8-gdb needn't builtin in stm-debug. Simply call shell command. Instalation of stm8-gdb may be job of user (dependency) no your job. OpenOCD same. SDCC same.
